### PR TITLE
13 improve branch coverage for importfromgeojsonimpl

### LINKED ISSUE #13: improve branch coverage for importFromGeoJsonImpl

### PR DESCRIPTION
**Pull request by WenJJ2000**
Added 4 test for importfromgeojsonimpl and improved branch coverage from 63% to 71%

Closes #13

### CODE REVIEW

File: src/main/java/com/esri/core/geometry/OperatorImportFromGeoJsonLocal.java [hunk 1]
```diff
@@ -240,8 +245,11 @@ MapOGCStructure importFromGeoJsonImpl(
             throw new JsonGeometryException(s);
           }
 
+          // System.out.println("initial geo json : " + geo_json_type);
           if (geo_json_type == GeoJsonType.GeometryCollection) {
             bCover.add(8);
+            // System.out.println("in branch 8");
+            // System.out.println(type);
```
Comment (LinusWallin) [inline src/main/java/com/esri/core/geometry/OperatorImportFromGeoJsonLocal.java:252]: remove commented code

**Review by LinusWallin — CHANGES_REQUESTED** [review]
commented code in pull request needs to be removed

**Comment by WenJJ2000** [comment]
done

**Review by LinusWallin — APPROVED** [review]
Lgtm

**Pull request merged**